### PR TITLE
docs(chore) redundant link to reuse doc -> moved to Attribution

### PR DESCRIPTION
**Pull request by bsmth**
The info in the content reuse doc is now contained within [Attribution and copyright licensing](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Attrib_copyright_license)

### CODE REVIEW

File: files/en-us/mdn/writing_guidelines/index.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ MDN Web Docs is an open-source project. The sections outlined below describe our
 
       - : Provides an in-depth criteria for content to be included on MDN Web Docs, the application process for getting new documentation added on MDN Web Docs, and the expectations and guidelines for a party applying.
 
-    - [Content reuse](/en-US/docs/MDN/Writing_guidelines/Content_reuse)
-      - : Describes our policy on content reuse and republishing, and how to get permission to republish content on MDN Web Docs.
+    - [Attribution and copyright licensing](/en-US/docs/MDN/Writing_guidelines/Attrib_copyright_license)
+      - : Describes our policy on using MDN Web Docs content elsewhere on the web, how to get permission to republish content on MDN, and hints for linking to MDN content.
```
Comment (dipikabh) [inline files/en-us/mdn/writing_guidelines/index.md:24]: - We should remove "Content reuse" here. "What we write" folder contains only the "criteria_for_inclusion" sub folder.
- "Attribution and copyright licensing" is covered as the last entry on this "Overview" page. Probably that description can be replaced with your text here.
- To match the TOC on the left, "How to label a technology" should ideally come after "Attribution and copyright licensing"

**Review by dipikabh — CHANGES_REQUESTED** [review]
- We should remove "Content reuse" here. "What we write" folder contains only the "criteria_for_inclusion" sub folder.
- "Attribution and copyright licensing" is covered as the last entry on this "Overview" page. Probably that description can be replaced with your text here.
- To match the TOC on the left, "How to label a technology" should ideally come after "Attribution and copyright licensing".

**Comment by bsmth** [comment]
Thanks Dipika, I think I've addressed your comments in https://github.com/mdn/content/pull/21631/commits/218badb5cf1ca977a03f0ff65b60e9d2f1c27183

**Review by dipikabh — APPROVED** [review]
Thank you! 👍

**Pull request merged**